executor/mock/standalone/standalone_test.go

```diff
@@ -1014,3 +1015,20 @@ func testSystemdImageMount(t *testing.T, jobID string) {
 		t.Fail()
 	}
 }
+
+// Test that the size of `/dev/shm` can be set
+func testShm(t *testing.T, jobID string) {
+	var mem int64 = 256
+	var shmSize int64 = 192
```

Comment:
What happens if `shmSize > mem`?

Reply:
This isn't relevant now that we're relying on the control plane to enforce this, but this is allowed.  The container will come up with a `/dev/shm` of whatever size you request.  However, you can only write to `/dev/shm` up to the cgroup memory limit of the container.